### DISCUSSION TITLE
Primefaces Ultima not compatible with JSF 2.2?

### DISCUSSION THREAD

**Discussion by erickdeoliveiraleal** [primefaces/community#2359] [PrimeFaces Templates] [answered]
I'm trying to integrate the Ultima (7.0.0) theme into my project (PrimeFaces 13.0.10)
However, I had an error when trying to copy the classes to my project in Eclipse, for example, line 25 of the InputStyleTypeConverter class has the code:
@FacesConverter(value = "org.primefaces.layout.inputStyleTypeConverter", managed = true)
However, from what I know, the managed = true part only works from JSF 2.3 onwards, so the included menu is not compatible with JSF 2.2?

Log error:  `The attribute managed is undefined for the annotation type FacesConverter`

**Comment by melloware** [comment, marked as answer]
@erickdeoliveiraleal i would just remove the managed="true" and recompile?  you have the source code right? Also as a note you know PF14 no longer supports JSF 2.2 so 13.0.10 will be the last version you can use of PF safely.

**Comment by tandraschko** [comment]
Plese see: https://x.com/primefaces/status/1970401400568918449
